src/Presets/northwestern-stubs/sass/nu_purple_layout.scss

```diff
@@ -87,7 +87,10 @@ footer ul li {
     height: 24px;
     width: 18px;
     margin-right: 1rem;
-    margin-left: -1rem;
```

Comment:
Should this be noted in the upgrade guide for dependents that've already published the styles?

Reply:
eh, it's only GSTS 🤷

Reply:
Fair, I wasn't sure if SOAP was going to eventually upgrade to BS5. Probably too much effort at this point.